RgueCmdr/ReadMe.txt

```diff
@@ -0,0 +1,30 @@
+RgueCmdr
+Smoothswing sound font for ProffieOS by Thexter.
+
+Modifications by Brian Conner 2022:
+Any additional sounds sourced from creative-commons licensed freesound.org.
+
+This ReadMe.txt added.
+Hierarchy made to be FAT32 friendly.
+
+Both .ini files have added link to their respective Wiki page for full descriptions and default values.
+
+config.ini edited.
+- Added "RgueCmdr config.ini"
+- humstart set to 636, other values use defaults.
+- Link to Wiki page added
+
+smoothsw.ini edited. 
+- Added "# RgueCmdr smoothsw.ini"
+- Link to Wiki page added
+
+bgnlock: Created from existing lock and additional sounds.
+
+lock: Edited to loop cleaner. Edited lock to have less monophonic-like hum mixed in and added some electrical arcing.
+
+hum: High passed @100Hz and normalized hum to -5db. This is in lieu of editing config.ini volHum parameter, as (initially) config.ini didn't exist. 
```

Comment:
I don't think we should highpass font files, since it's possible to hook up a Proffieboard to an external amplifier that can produce lots of base.

Reply:
How about if we include "original_hum.wav" for that purpose? IMO the vast majority of users are using insaber speakers, and the objective was to make it sound better for typical use.
Else, indicate somewhere that it's suggested to use  
`#define FILTER_CUTOFF_FREQUENCY 100  
#define FILTER_ORDER 8`  
?